api/catalog/api/controllers/search_controller.py

```diff
@@ -340,18 +340,25 @@ def search(
     search_fields = ["tags.name", "title", "description"]
     if "q" in search_params.data:
         query = _quote_escape(search_params.data["q"])
+        base_query_kwargs = {
+            "query": query,
+            "fields": search_fields,
+            "default_operator": "AND",
+        }
+
+        if '"' in query:
+            base_query_kwargs["quote_field_suffix"] = ".exact"
+
         s = s.query(
             "simple_query_string",
-            query=query,
-            fields=search_fields,
-            default_operator="AND",
+            **base_query_kwargs,
         )
-        # Boost exact matches
+        # Boost exact matches on the title
         quotes_stripped = query.replace('"', "")
         exact_match_boost = Q(
             "simple_query_string",
-            fields=["title"],
-            query=f'"{quotes_stripped}"',
+            fields=["title.exact"],
```

Comment:
I think this might actually be breaking the exact match boost, oddly. When searching without quotes (http://localhost:50280/v1/images/?q=bird%20perched), I noticed that some results that did **not** have exact title match were included in the first page of results. (For example, there's a `Black Bird Photo` and `Bird Nature Photo` mixed in.)

I tried removing the `.exact` added here and then the boost appeared to work again -- only photos titled `Bird Perched Photo` appeared in the first page of results.

Reply:
Interesting! Thank you for looking into and testing this. I'll make the change (and do a little reading to understand why that would be the case :thinking:)

Reply:
It is very weird! I tried removing the `.exact` totally on a whim to compare, definitely expected your syntax here to be the correct one 🤔

Reply:
Staci, I did a bit of reading in the ES documentation because I was confused why it didn't work the way we expected. I think it comes down to us using the `bool` query in a particular way that I have to admit, I do not fully understand. I tried to re-write the boost (just out of curiosity, not for this PR) to use the field-boosting described in the simple query string DSL documentation, but I could not get the results to budge _at all_. I am pretty curious to read more about Elasticsearch to understand better how these things are meant to work and how score boosting is best approached. It'd be nice, in any case, to document the current approach, why it works and why (if it indeed is) it is the best and correct approach for our use case.

Anyway, I removed the `.exact` on this one and things are back in working order. Thanks again for looking into this!

Reply:
Thanks for looking into it! Super strange.

I re-tested and everything looks good to me except for the test failure. Not sure what's going on there, as. when I test locally it's definitely the case that we get more unquoted results than quoted ones.